javascript/exercises/credit_card_validator.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```

Comment:
This is extending on an exercise in the javascript directory (the credit card validator exercise from JS1 weekend).